### DISCUSSION TITLE
Invaild requests

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [community/community#12995] [Copilot Conversations] [answered]
I have problem while activate GitHub Copliot (Error: GitHub Copilot could not connect to server. Extension activation failed: "User not authorized")

I found this discussions https://github.com/github/feedback/discussions/12812, https://github.com/github/feedback/discussions/9342, 
https://github.com/github/feedback/discussions/7707. And I found out this (Bad requests: 20.205.243.168 (to api.github.com for copliot services tokens))

What even that mean????

![Screenshot 2022-03-18 124000](https://user-images.githubusercontent.com/95196459/158944143-4b109f17-1638-47fa-b084-08408a13887a.png)

**Comment by davecheney** [comment, marked as answer]
@DinhHuy2010 it looks like you are still on the waitlist. I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Reply by DinhHuy2010** [reply]
ok
